components/HomeProducts.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect} from 'react'
+import { Flex, Grid, Box, Text} from '@chakra-ui/react'
+import Product from './Product'
+
+function HomeProducts({coffees, itemsLeft}) {
+  const [coffeesArr, setCoffeeArr] = useState([...coffees])
+  const [coffeesLeft, setCoffeesLeft] = useState(itemsLeft)
+  const [query, setQuery] = useState(false)
```

Comment:
A good convention with boolean variables is to write as verbs or questions like isFecthing isLoading, even fetching or loading can work too.

Reply:
I will change it to `isLoading`, thanks

Reply:
Sure, thank to you